api/db/load_vars.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
I'd add a line or two of comments here, providing context on what this script is and when / from where it should expect to be run. Just for posterity.